app/vmctl/flags.go

```diff
@@ -29,6 +29,32 @@ var (
 	}
 )
 
+const (
+	backoffRetries     = "backoff-retries"
+	backoffFactor      = "backoff-factor"
+	backoffMinDuration = "backoff-min-duration"
+)
+
+var (
+	retryFlags = []cli.Flag{
```

Comment:
From the code below, looks like backoff flags are only used by VM importer and VM native. So, I think, these flags should belong to `vmFlags`.